# Patch for 3 dimensions instead of the standard 2
# Contribution from the Crick HPC team
# uploaded by J. Sassmannshausen
diff -Nru cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b-orig/src/math/dim.h cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b/src/math/dim.h
--- cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b-orig/src/math/dim.h	2019-01-17 09:14:25.000000000 +0000
+++ cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b/src/math/dim.h	2019-02-25 10:17:50.224521000 +0000
@@ -7,6 +7,6 @@
 #ifndef DIM
 
 /// DIM defines the number of dimensions in space: 1, 2 or 3
-#define DIM 2
+#define DIM 3
 
 #endif
